ballerina/Dependencies.toml

```diff
@@ -1,127 +0,0 @@
-# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Any need to remove Dependencies.toml?

Reply:
Has been mistakenly committed. Added again in [004bb4d](https://github.com/ballerina-platform/module-ballerina-persist/pull/82/commits/004bb4dd7e1d189b672413c7fe92245486ffdf4d)